internal/pkg/aws/cloudwatch/cloudwatch.go

```diff
@@ -101,6 +101,31 @@ func (cw *CloudWatch) AlarmStatus(alarms []string) ([]AlarmStatus, error) {
 	return alarmStatus, nil
 }
 
+// AlarmStatusesFromNamePrefix returns the status of all alarms whose names begin with the given prefix.
+func (cw *CloudWatch) AlarmStatusesFromNamePrefix(prefix string) ([]AlarmStatus, error) {
```

Comment:
Can we remove duplication with `AlarmStatus` by defining

```go
func (cw *CloudWatch) AlarmStatus(options ...AlarmStatusOption) ([]AlarmStatus, error)
```

and caller can call either
```go
cw.AlarmStatus(WithNames(names))
```
or
```go
cw.AlarmStatus(WithPrefix(fmt.Sprintf("%s-%s-%s-CopilotRollback", app, env, svc)))